Source/LightboxViewCell.swift

```diff
@@ -4,10 +4,43 @@ public class LightboxViewCell: UICollectionViewCell {
 
   public static let reuseIdentifier: String = "LightboxViewCell"
 
+  var constraintsAdded = false
+
   public lazy var lightboxView: LightboxView = { [unowned self] in
     let lightboxView = LightboxView(frame: self.bounds)
+    lightboxView.setTranslatesAutoresizingMaskIntoConstraints(false)
+
     self.contentView.addSubview(lightboxView)
 
     return lightboxView
   }()
+
+  public override func layoutSubviews() {
+    super.layoutSubviews()
+
+    setupConstraints()
+    lightboxView.updateViewLayout()
+  }
+
+  private func setupConstraints() {
+    if !constraintsAdded {
+      addConstraint(NSLayoutConstraint(item: lightboxView, attribute: .Leading,
```

Comment:
@zenangst @richardoti Exactly, in Lightbox we don't have any dependency right now.